babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    presets: ['@babel/preset-react', '@babel/preset-env'],
+}
```

Comment:
We should add Emotion and make sure we configure both theses Babel plugins.

We should also cover Jest in the Babel config as well. Make sure it's outputting `commonjs` modules.

One example is how in development mode, we're adding React FastRefresh.

The Babel config can accept an `env` prop at the top and there you can define `development`, `production`, and `test`. We do this on some other projects at Minted.

Here's an example of the presets I put in Addressbook:
```js
  presets: [
    [
      '@babel/preset-env',
      {
        modules: (
          isTest
            ? 'commonjs'
            : false
        ),
        useBuiltIns: false,
      },
    ],
    [
      '@babel/preset-react',
      {
        importSource: '@emotion/react',
        runtime: 'automatic',
      },
    ],
    [
      '@emotion/babel-preset-css-prop',
      emotionBabelPluginConfig,
    ],
  ],
```

Please don't copy-paste as this project does not have the environment config separation.

Reply:
what do you mean by `environment config separation.`? can we not just set const isTest = process.env.NODE_ENV === 'test';` at the top and then use the variable in the config?

Reply:
You can do that, but it's far easier for someone to read if it's `{ env: { development: {}, production: {}, test: {} }`.